packages/driver/cypress/e2e/commands/actions/type_special_chars.cy.ts

```diff
@@ -557,7 +568,9 @@ describe('src/cy/commands/actions/type - #type special chars', () => {
       cy.get('[contenteditable]:first').invoke('text', 'ab')
       .type('{moveToStart}')
       .then(($el) => {
```

Comment:
what type winds up coming through here, `HTMLElement`? Wonder if typing here prevents the unknown?

Reply:
Yah, just an HTMLElement, but also we don't know! so you have to convert to unknown first. In order to call `.getSelection`, it HAS to know that it's an HTMLInputElement. I almost feel like our own cypress types could be better about the type that it's passing through chaining that could help with this, but that sounds complex.

Reply:
Yeah that sounds like a pain to do here. We really should improve the yielded types in these cases when we can